language-support/ts/daml-ledger/index.ts

```diff
@@ -154,12 +154,12 @@ class Ledger {
   /**
    * Create a contract for a given template.
    */
-  async create<T extends object, K>(template: Template<T, K>, argument: T): Promise<CreateEvent<T, K>> {
-    const payload = {
+  async create<T extends object, K>(template: Template<T, K>, contractPayload: T): Promise<CreateEvent<T, K>> {
```

Comment:
```suggestion
  async create<T extends object, K>(template: Template<T, K>, payload: T): Promise<CreateEvent<T, K>> {
```

Seems more in the spirit of the change, no?